src/cannon-es-debugger-babylonjs.ts

```diff
@@ -10,6 +10,8 @@ import {
   Quaternion,
   StandardMaterial,
   Vector3,
+  Mesh as BabylonMesh,
```

Comment:
So far I had convention of aliasing `cannon's` names so that can be just `Mesh`.
Also please run `yarn lint` locally and fix issues.
I will add lint check for the repo tomorrow with #44